draft-ietf-quic-transport.md

```diff
@@ -322,175 +322,356 @@ all field sizes are in bits.  When discussing individual bits of fields, the
-     is sent by the server or the client.  A client MAY set this flag and
-     include exactly one proposed version.  A server may set this flag when the
-     client-proposed version was unsupported, and may then provide a list (0 or
-     more) of acceptable versions as a part of version negotiation (described in
-     {{version-negotiation}}.)
+Header Form:
 
-   * 0x02 = PUBLIC_RESET.  Set to indicate that the packet is a Public Reset
-     packet.
+: The most significant bit (0x80) of the first octet (octet 0) is set to 1 for
+  long headers and 0 for short headers.
 
-   * 0x04 = KEY_PHASE.  This is used by the QUIC packet protection to identify
-     the correct packet protection keys, see {{QUIC-TLS}}.
+Long Packet Type:
 
-   * 0x08 = CONNECTION_ID.  Indicates the Connection ID is present in the
-     packet.  This must be set in all packets until negotiated to a different
-     value for a given direction.  For instance, if a client indicates that the
-     5-tuple fully identifies the connection at the client, the connection ID is
-     optional in the server-to-client direction. The negotiation is described in
-     {{transport-parameter-definitions}}.
+: The remaining seven bits of octet 0 of a long packet is the packet type.  This
+  field can indicate one of 128 packet types.  The types specified for this
+  version are listed in {{long-packet-types}}.
 
-   * 0x30 = PACKET_NUMBER_SIZE.  These two bits indicate the number of
-     low-order-bytes of the packet number that are present in each packet.
+Connection ID:
 
-     + 11 indicates that 6 bytes of the packet number are present
-     + 10 indicates that 4 bytes of the packet number are present
-     + 01 indicates that 2 bytes of the packet number are present
-     + 00 indicates that 1 byte of the packet number is present
+: Octets 1 through 8 include a connection ID.  {{connection-id}} describes the
+  use of this field in more details.
 
-   * 0x40 = MULTIPATH.  This bit is reserved for multipath use.
+Packet Number:
 
-   * 0x80 is currently unused, and must be set to 0.
+: Octets 9 to 12 contain a packet number.  {{packet-numbers} describes the use
+  of packet numbers.
 
-* Connection ID: An unsigned 64-bit random number chosen by the client, used as
-  the identifier of the connection.  Connection ID is tied to a QUIC connection,
-  and remains consistent across client and/or server IP and port changes.
+Version:
 
+: Octets 13 to 16 contain the selected protocol version.  This field indicates
+  which version of QUIC is in use and determines how the some protocol fields
+  are interpreted.
 
-### Identifying Packet Types
+Payload:
 
-While all QUIC packets have the same common header, there are three types of
-packets: Regular packets, Version Negotiation packets, and Public Reset packets.
-The flowchart below shows how a packet is classified into one of these three
-packet types:
+: Octets from 17 onwards are the payload of the packet.
```

Comment:
I've added "(the rest of the QUIC packet)" for consistency since we don't really need to talk about UDP datagram anywhere else after QUIC packet is defined.